design_wiki/saplings/Index.md

```
@@ -0,0 +1 @@
+This is the `saplings` directory, for design notes more mature than `seeds` but not yet ready to be moved to the design document.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)